src/network.rs

```diff
@@ -123,12 +123,12 @@ impl<H: Hasher, D: Data, S: Signature, MS: PartialMultisignature, N: Network<H,
         match recipient {
             Everyone => {
                 if let Err(error) = self.network.broadcast(data) {
-                    error!(target: "AlephBFT-etwork-hub", "Broadcast error: {:?}", error);
+                    error!(target: "AlephBFT-network-hub", "Broadcast error: {:?}", error);
```

Comment:
Oh no, this will be a conflict with the logging PR where I also do it...

Reply:
:(